depends/common/2048/CMakeLists.txt

```diff
@@ -17,13 +17,22 @@ elseif("${CORE_SYSTEM_NAME}" STREQUAL "linux")
 elseif("${CORE_SYSTEM_NAME}" STREQUAL "darwin" OR "${CORE_SYSTEM_NAME}" STREQUAL "osx")
   set(BUILD_COMMAND make -f Makefile.libretro platform=osx ${LIBRETRO_DEBUG})
 elseif("${CORE_SYSTEM_NAME}" STREQUAL "ios")
+  # TODO: Should we use platform=osx?
```

Comment:
Looking at this: https://github.com/libretro/snes9x/blob/master/libretro/Makefile
it should be 'ios'. Rbpi and android arm should just include include 'armv' from what i can see.